timebasedcv/splitstate.py

```diff
@@ -115,3 +120,21 @@ def total_length(self: Self) -> timedelta:
             A `timedelta` object representing the time between `train_start` and `forecast_end`.
         """
         return self.forecast_end - self.train_start
+
+    def __add__(self: Self, other: Union[timedelta, pd.Timedelta]) -> SplitState:
```

Comment:
Ended up not using this as for the `window="rolling"` case it would break and be handled anyway, yet I don't mind to keep it